apps/storybook.namekit.io/stories/Namekit/TruncatedText.stories.tsx

```diff
@@ -1,44 +1,36 @@
 import { TruncatedText } from "@namehash/namekit-react/client";
 import type { Meta, StoryObj } from "@storybook/react";
 
-enum TruncatedTextExample {
-  SHORT_TEXT = "SHORT_TEXT",
-  LONG_TEXT = "LONG_TEXT",
-}
-
-const getExampleTruncatedText = (example: TruncatedTextExample): string => {
-  switch (example) {
-    case TruncatedTextExample.SHORT_TEXT:
-      return "heyiamsmall";
-    case TruncatedTextExample.LONG_TEXT:
-      return "heyiamaverylongtextthatkeepsgrowingunstopablyomgiamstillgoingonokdone";
-  }
-};
+const SHORT_TEXT = "heyiamsmall";
+const LONG_TEXT = "heyiamaverylongtextthatkeepsgrowingunstopablyomgiamstillgoingonokdone";
 
 const meta: Meta<typeof TruncatedText> = {
   component: TruncatedText,
   title: "Namekit/TruncatedText",
   argTypes: {
-    textStylingClasses: {
-      control: { type: "text" },
-    },
-    tooltipTextStylingClasses: {
+    text: {
       control: { type: "text" },
     },
     maxDisplayWidth: {
       control: { type: "number" },
     },
+    textStylingClasses: {
+      control: { type: "text" },
+    },
+    displayTooltipWhenTextOverflows: {
+      control: { type: "boolean" },
+    },
     maxTooltipWidth: {
       control: { type: "number" },
     },
+    tooltipTextStylingClasses: {
+      control: { type: "text" },
+    },
   },
   args: {
```

Comment:
I fully aligned the default args here with the default args that actually exist on `TruncatedText`. I think this is the right strategy. Therefore, it is correct that a field such as `maxTooltipWidth` should be undefined in storybook by default.